test/u0_n_parameter_inputs.jl

```diff
@@ -6,10 +6,10 @@ include("test_networks.jl")
 
 # Tests for reaction_networks_standard[7]
 test_network = reaction_networks_standard[7]
-@variables X1 X2 X3 X4 X5 X6 X
-@parameters p1 p2 p3 k1 k2 k3 v1 K1 d1 d2 d3 d4 d5
+@parameters t p1 p2 p3 k1 k2 k3 v1 K1 d1 d2 d3 d4 d5
+@variables X1(t) X2(t) X3(t) X4(t) X5(t) X6(t) X(t)
```

Comment:
@isaacsas this is the only change we made after you left the call.

Reply:
Great, thanks!

Reply:
I’ll test the graph and latexify code tomorrow and then make a release. (Tests are disabled for those currently.)